include/snowflake/version.h

```diff
@@ -5,6 +5,6 @@
 #ifndef SNOWFLAKE_CLIENT_VERSION_H
 #define SNOWFLAKE_CLIENT_VERSION_H
 
-#define SF_API_VERSION "1.0.10"
+#define SF_API_VERSION "1.0.10.1"
```

Comment:
why do we bump the version here? cannot we use 1.0.10?

Reply:
changed back.